src/esp/gfx/LightSetup.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include "LightSetup.h"
+
+namespace esp {
+namespace gfx {
+
+Magnum::Vector3 getLightPositionRelativeToCamera(
+    const LightInfo& light,
+    const Magnum::Matrix4& transformationMatrix,
+    const Magnum::Matrix4& cameraMatrix) {
+  if (light.model == LightPositionModel::OBJECT)
+    return transformationMatrix.transformPoint(light.position);
+
+  if (light.model == LightPositionModel::GLOBAL)
+    return cameraMatrix.transformPoint(light.position);
+
+  // LightPositionModel::CAMERA
+  return light.position;
+}
+
+LightSetup getLightsAtSceneCorners(scene::SceneGraph& sceneGraph,
+                                   Magnum::Float lightIntensity) {
+  const Magnum::Range3D& sceneBB =
+      sceneGraph.getRootNode().computeCumulativeBB();
```

Comment:
If we intend to expose this to the user or call it again after initialization we may want to pre-compute this bounding box after initial load and pass it in instead. `computeCumulativeBB()` on the root will recurse the full SceneGraph. This is fine to do once, but could be trouble in some edge cases (e.g. an object falls through a hole and stretches the root bounding box on recompute or a user calls this function every other frame for some reason.)

This could be `getLightsAtBoxCorners` or similar and take a `Magnum::Range3D` instead. Would let users place lights for arbitrary bounding regions. Could include an optional (default Identity) transformation parameter also to provide non-axis aligned box support.

Reply:
left out the transformation parameter to keep it simple, if user wants they can transform range before.